src/pages/About.js

```diff
@@ -1,11 +1,36 @@
 import React, { Component } from 'react';
 import { Route } from 'react-router-dom';
 import Toolbar from '../components/Toolbar.js';
+import { Container, Col, Row } from 'react-bootstrap';
+import '../css/About.css';
+import Members from '../components/Members.js';
 
 class About extends Component {
 	render() {
 		return (
-			<h1> About </h1>
+			<Container>
+			<div className="div">
+			<Row>
+			<Col>
+				<h1 className="Title"> Our Mission </h1>
+			</Col>
+			<Col>
+				<h1 className="Content1"> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor </h1>
+				<h2 className="Content2"> incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam. Aliquet tristique nisl vitae volutpat. Nulla aliquet porttitor venenatis.</h2>
+				<h3 className="Content3"> Donec a dui et dui fringilla consectetur id nec massa. Aliquam erat volutpat. Sed ut dui ut lacus dictum fermentum vel tincidunt neque. Sed sed lacinia lectus. Duis sit amet sodales felis. Duis nunc eros, mattis at dui ac, convallis semper risus. In adipiscing ultrices tellus, in suscipit massa vehicula eu.</h3>
+			</Col>
+			</Row>
+			<Row>
+				<Col>
+					<h1 className="Content4"> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</h1>
+				</Col>
+				<Col>
+					<h1 className="Title2"> The Lineup </h1>
+				</Col>
+			</Row>
+				<Members/>
```

Comment:
add 2 or 3 newline's using <[br]/> (without the brackets, I added them bc github adds a newline instead of the text) at the end of the container so that there's some padding between a member and the bottom of the screen.